package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "movies-app",
+  "version": "1.0.0",
+  "description": "A database application for movie info & trivia",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

start script is missing. You could add: 
```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "NODE_ENV=prod node server need to be specified",    
    "dev": "nodemon server"
```
then to running it you could type `npm run dev`

or you could have `"start": "node app.js"` if you prefer